extensions/tana/README.md

```diff
@@ -8,4 +8,21 @@ Write a note in Raycast and and send it to Tana! The note that you write will be
 
 To configure this in Raycast, you'll need to provide an API token which can be generated in Tana. To generate this token, open the command line (Cmd/Ctrl-K) and select `Get API Token`. This will generate a token and add it your clipboard. Once generated, paste the token into the corresponding configuration in Raycast.
```

Comment:
This is outdated now with the move to addToNodeV2. Can this readme be updated with instructions on how to update the legacy API token with the new workspace API token? Both on the Raycast side and the Tana side. This should also let users know the note will appear in their Inbox by default.